paywall/src/__tests__/utils/validators.test.js

```diff
@@ -1037,10 +1066,31 @@ describe('Form field validators', () => {
             outstandingKeys: 0,
             balance: '0',
             owner: '0x1234567890123456789012345678901234567890',
+            publicLockVersion: 4,
+            currencyContractAddress:
+              '0x9876543210987654321098765432109876543210',
+            some: 'random',
```

Comment:
here is the test that makes sure if we add new fields in the future it still is valid